packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte

```diff
@@ -79,14 +79,16 @@ $: resetToDefault = false;
         showUpdate="{false}"
         record="{recordUI.original}"
         updateResetButtonVisibility="{updateResetButtonVisibility}"
-        resetToDefault="{resetToDefault}" />
+        resetToDefault="{resetToDefault}"
+        enableAutoCompletion="{true}" />
```

Comment:
```suggestion
        enableAutoCompletion="{true}" />
```

probably need to update the autocompletion to autoSave